test/cache_test.rb

```diff
@@ -144,6 +149,28 @@ class InheritedRoleSerializer < RoleSerializer
       @blog_serializer     = BlogSerializer.new(@blog)
     end
 
+    def test_expiring_of_cache_at_update_of_record
+      if ARModels::Author.respond_to?(:cache_versioning)
+        ARModels::Author.cache_versioning = true
+      end
```

Comment:
This fails without the cache versioning? why?

Reply:
😅 I think I specified it in the description :P 

It fails because the current cache implementation rely on the `cache_key` provided by ActiveRecord to contains the `timestamp` but when `cache_versioning` is true the key of course does not contain the `timestamp`. 

`cache_versioning` changes the strategy of caching for rails implementing "reusable keys". This means that the key never change, and the content of the cache instead contains the version (timestamp in this case).